internal/storage/metrics.go

```diff
@@ -0,0 +1,8 @@
+package storage
+
+type Metrics struct {
```

Comment:
[CLARIFICATION] Why is it here?
[CLARIFICATION] What is it? DTO or the domain object?